waddrmgr/db.go

```diff
@@ -1268,8 +1308,41 @@ func upgradeManager(namespace walletdb.Namespace) error {
 
 	// Upgrade the manager as needed.
 	if version < LatestMgrVersion {
-		// No upgrades yet.
+		// Upgrade addresses used flag
+		upgradeVersion1to2(namespace)
 	}
 
 	return nil
 }
+
+// upgradeVersion1to2 upgrades the database from version 1 to version 2
+// 'usedAddrBucketName' a bucket for storing addrs flagged as marked is initialized
+// and it will be updated on the next rescan
+func upgradeVersion1to2(namespace walletdb.Namespace) error {
+	err := namespace.Update(func(tx walletdb.Tx) error {
+		rootBucket := tx.RootBucket()
+		mainBucket := tx.RootBucket().Bucket(mainBucketName)
+
+		_, err := rootBucket.CreateBucketIfNotExists(usedAddrBucketName)
```

Comment:
OK, I thought we might need this for migrating existing db but I guess `upgradeManager` takes care of that. If we don't need this, should we keep the existing version number since we don't need any upgrades?